test/test_sparse_csr.py

```diff
@@ -396,6 +394,34 @@ def test_csr_matvec(self, device, dtype):
             with self.assertRaisesRegex(RuntimeError, "mv: expected"):
                 csr.matmul(bad_vec)
 
+    @onlyCUDA
+    @dtypes(torch.double)
+    def test_mm(self, device, dtype):
+        def test_shape(di, dj, dk, nnz):
+            x = self.genSparseCSRTensor((di, dj), nnz, device=device, dtype=dtype, index_dtype=torch.int32)
```

Comment:
For completeness, CSR tensors should be tested for both int32 and int64 indices.